src/partials/layout/_footer.html

```diff
@@ -0,0 +1,14 @@
+<footer class="footer">
+  <div class="container">
+    <div class="footer-wrapper">
+      <p class="footer-wrapper__text">© 2020 | All Rights Reserved |</p>
```

Comment:
Rename class to footer-wrapper__copyright

Reply:
Renamed footer-wrapper__text to footer-wrapper__copyright.